.browserslistrc

```diff
@@ -0,0 +1,5 @@
+ie >= 10
+last 2 versions
```

Comment:
careful with `last 2 versions` these are and clauses. Compare these 2
http://browserl.ist/?q=ie+%3E%3D+10%2C+last+2+android+versions%2C+last+2+and_chr+versions%2C+iOS+%3E%3D+10
vs
http://browserl.ist/?q=ie+%3E%3D+10%2C+last+2+android+versions%2C+last+2+and_chr+versions%2C+iOS+%3E%3D+10%2C+last+2+versions

last 2 versions gets you awesome browsers like Blackberry 7

https://jamie.build/last-2-versions

Reply:
Thanks for the info @alex-bezek, I wouldn't have considered that. Nice test site too.

Reply:
Yikes, so this has been a problem for awhile.